src/main/resources/application.properties

```diff
@@ -20,3 +20,5 @@ sam.basePath=${SAM_ADDRESS}
 
 datarepo.instances.terra-dev=https://jade.datarepo-dev.broadinstitute.org
 datarepo.instances.terra-prod=https://jade-terra.datarepo-prod.broadinstitute.org
+
+logging.pattern.level = %X{requestId}%5p
```

Comment:
I think if you change this to `%X{requestId} %5p` (note the space) then you won't have to append a space to the requestid in `MdcLogEnhancerFilter`. See https://docs.spring.io/autorepo/docs/spring-boot/1.4.0.RELEASE/reference/html/boot-features-logging.html (search for "You can add MDC and other ad-hoc content")